src/python/pants/backend/jvm/tasks/coursier_resolve.py

```diff
@@ -146,11 +146,11 @@ def resolve(self, targets, compile_classpath, sources, javadoc):
         pants_jar_base_dir = self._prepare_workdir(pants_workdir)
         coursier_cache_dir = CoursierSubsystem.global_instance().get_options().cache_dir
 
-        # Check each individual target without context first
-        if not invalidation_check.invalid_vts:
-
+        # If a report is requested, do not proceed with loading validated result.
```

Comment:
It would be good to add a test of the report case, as it broke accidentally in ivy, and seems liable to break here as well.

Reply:
Good idea. Done.